src/backend/partaj/core/models/referral_close_history.py

```diff
@@ -0,0 +1,44 @@
+"""
+Referral close history model in our core app.
+"""
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ReferralCloseHistory(models.Model):
```

Comment:
I liked your idea of a `ReferralActivityMessage` object that we can use for all activities that just need to hold one text field for a given activity. I think we encountered another use-case during our roadmap meeting.

However,, now I think that could be done by just adding a generic `message` field to `ReferralActivity`. This way we avoid adding a join for one field and keep the flexibility of not holding this information on the `Referral` itself.

What do you think?

Reply:
It's a good idea. I will do it